src/api/RestAPI.ts

```diff
@@ -288,21 +258,23 @@ export class RestAPI extends EventEmitter {
             headers.append("Content-Type", "application/json");
         }
 
-        const origin = getOrigin(data) || this.origin;
+        const {
+            origin = this.origin,
+            idempotentKey,
```

Comment:
If you want to follow the old behavior you may want to default assign it to `null` 🙂

Reply:
ok sure, fixed! 🙇